packages/rancher-external-ip-webhook/package.yaml

```diff
@@ -1,3 +1,3 @@
 url: https://github.com/rancher/externalip-webhook/releases/download/v1.0.1/rancher-externalip-webhook-1.0.1.tgz
-version: 101.0.0
+version: 100.0.2
 doNotRelease: false
```

Comment:
@KevinJoiner I talked to Prachi and they want to remove the package.yaml for removed charts.